src/sentry/static/sentry/app/types/iconProps.ts

```diff
@@ -0,0 +1,8 @@
+export interface IconProps extends React.SVGAttributes<SVGElement> {
+  color?: string;
```

Comment:
Should `color` and `size` be required since they all have default params set? `React.FC` is broken and will complain so we may need to remove it

Reply:
What should I replace React.FC with?

Reply:
Yes, you're right - color and size should be required. Nice catch

Reply:
Ooops, ignore the `React.FC` suggestion, it only doesn't work if you use `defaultProps`;